src/server/rest/v1/service/RegistrationTokenService.ts

```diff
@@ -58,6 +58,16 @@ export default class RegistrationTokenService {
         user: req.user
       });
     }
+    // Check site is provided for site admins
+    if (Authorizations.isSiteAdmin(req.user) && !filteredRequest.siteAreaID) {
+      throw new AppError({
```

Comment:
Use AppAuthError exception for authorization

Reply:
done

Reply:
Do not mark the conversation solved yourself (golden rule 44)